### ISSUE TITLE
UNet on WH 

### ISSUE THREAD

**Issue by dvartaniansTT** [tenstorrent-metal/tt-metal#5924]
**Is your feature request related to a problem? Please describe.**
run UNet on WH - all ops should run on device. 

**Describe the solution you'd like**
replicate the same functionality as in gs.

**Comment by nsmithtt** [comment]
Gah, sorry I accidentally filed this one: https://github.com/tenstorrent-metal/tt-metal/issues/6076

**Comment by nsmithtt** [comment]
Comments from the other issue:

Initial repro instructions

```
git checkout tpatel/unet-wh
python tests/ttnn/integration_tests/unet/test_ttnn_shallow_unet.py
```

Member
[tapspatel](https://github.com/tapspatel) commented [1 hour ago](https://github.com/orgs/tenstorrent-metal/projects/81/views/1?pane=issue&itemId=55583607#issuecomment-1981223181) • 
edited 
need to disable shallow Conv optimization as its not functional on wormhole yet @tt-nshanker
 @tapspatel

Member
[tapspatel](https://github.com/tapspatel) commented [51 minutes ago](https://github.com/orgs/tenstorrent-metal/projects/81/views/1?pane=issue&itemId=55583607#issuecomment-1981233048)
hit this assert next
```
assert out_block_h_ntiles % act_block_h_ntiles == 0, "act_block_h must evenly divide out_block_h"
AssertionError: act_block_h must evenly divide out_block_h
need to refactor current overrides for WH - currently they are GS specific
```

relevant code in determine_per_core_block_config in tt_eager/tt_dnn/op_library/sliding_window_op_infra/tt_py_composite_conv.py
 @tapspatel

Member
[tapspatel](https://github.com/tapspatel) commented [28 minutes ago](https://github.com/orgs/tenstorrent-metal/projects/81/views/1?pane=issue&itemId=55583607#issuecomment-1981287347) • 
edited 
Disabling any user config overrides since they were specific to GS opts

```
#if "act_block_h" in config_override:
    #    act_block_h_override = config_override["act_block_h"]
    #    assert act_block_h_override % 32 == 0, "act_block_h must be divisible by 32 (tile height)"
```
another change that must be made is the padded input channels into c1 should be changed to 32
```
ttnn_module_args.c1["padded_input_channels"] = 32
```
this takes us to the next error on c1
```
Statically allocated circular buffers on core range {} grow to {} B which is beyond max L1 size of {} B
[(x=0,y=0) - (x=7,y=7)]
1679588
1499136
```

**Comment by tt-nshanker** [comment]
@tapspatel, The error on WH is expected since the override configs are set based on GS. I have discussed with @dvartaniansTT how to determine the the new override numbers (if they are required for WH). Let me know if you want to make the changes and I can show you.

**Comment by dvartaniansTT** [comment]
@tapspatel and @nsmithtt should we close this one or merge it to main first?

**Comment by nsmithtt** [comment]
@dvartaniansTT / @tapspatel , let's wait to close until it's merged to main

**Comment by tapspatel** [comment]
Merged into main: e32597bbfceae88660bf32bf5abe2eacca4953a2. Closing issue
